site/visitor-console/src/library/constants.ts

```diff
@@ -7,9 +7,40 @@ export const api_endpoint =
 export const format_date = (date: Date) => date.toISOString().split("T")[0];
 
 export const locations: MakerspaceLocation[] = [
-  { slug: "cooper_library", name: "Cooper Library" },
-  { slug: "watt_center", name: "Watt Family Innovation Center" },
-  { slug: "cook_lab", name: "Cook Laboratory" },
+  {
+    slug: "watt_center",
+    name: "Watt Family Innovation Center",
+    tools: [
+      "3D Print (FDM - Plastic)",
```

Comment:
`3D print` and `Resin print` should probably both be “Printer” not “Print”

Reply:
Revised the items to refer to the tools themselves and not any action associated with the tool; like a "print" of a "3d printer".